RELEASENOTES/v2.0.0.alpha-4.MD

```diff
@@ -0,0 +1,58 @@
+# Notary v2 Alpha 4 Release
+
+## Overview
+Notary v2 working group is pleased to announce the fourth release, Alpha 4. Refer release definitions [here](https://github.com/notaryproject/notation/blob/main/RELEASE_MANAGEMENT.md). This release includes updates to the [notaryproject/notaryproject](https://github.com/notaryproject/notaryproject) repo to provide the community with updated specifications and an updated implementation of Notation client for signing and verifying artifacts.
+
+- [Notary v2 specifications](https://github.com/notaryproject/notaryproject) ***[(notaryproject-v1.0.0-draft.4)](https://github.com/notaryproject/notaryproject/releases/tag/v1.0.0-draft.4)***    (Updated in this release)  
+- [Notation Library 1](https://github.com/notaryproject/notation-core-go) ***[(notation-core-go-v0.1.0-alpha.4)](https://github.com/notaryproject/notation-core-go/releases/tag/v0.1.0-alpha.4)***   (Updated in this release)
+- [Notation Library 2](https://github.com/notaryproject/notation-go) ***[(notation-go-v0.10.0-alpha.4)](https://github.com/notaryproject/notation-go/releases/tag/v0.10.0-alpha.4)***   (Updated in this release)
+- [Notation CLI](https://github.com/notaryproject/notation)   ***([notation-v0.10.0-alpha.4)](https://github.com/notaryproject/notation/releases/tag/v0.10.0-alpha.4)***   (Updated in this release) 
```

Comment:
The version for notation cli is v0.11.0-alpha.4

Reply:
Done